Orange/statistics/util.py

```diff
@@ -316,3 +316,13 @@ def unique(x, return_counts=False):
         if explicit_zeros:
             return r
         return np.insert(r, 0, 0)
+
+
+def var(x, axis=0):
```

Comment:
To be consistent with numpy and the methods above, I suggest we set the default value for axis as `None` instead of `0`.